acceptance/tests/environment/can_enumerate_environments.rb

```diff
@@ -56,7 +56,7 @@ def curl_master_from(agent, path, headers = '', &block)
 with_puppet_running_on(master, master_opts) do
   agents.each do |agent|
     step "Ensure that an unauthenticated client cannot access the environments list" do
-      on agent, "curl --tlsv1 -ksv https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
+      on agent, "curl --tlsv1 -s -v https://#{master}:#{master_port(agent)}/puppet/v3/environments", :acceptable_exit_codes => [0,7] do
```

Comment:
Hmm, nope. Looks like I only solved it in one place - `wait_for_host_in_dashboard` in beaker, and not in any generalized curl helper :(

There is a `curl_on` helper in Beaker that could be fixed to handle the -k case, though